src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1708,71 +1708,128 @@ Below is an example for adding a certificate to a Java cacerts file:
+The view of workflows from Catalog in Studio and from Workflow Catalog Portal follow the same view logic. The page is separated in 3 sections.
+The approach is selecting the needed information from left to right. First user should choose the required bucket, then the user can observe all the workflows in the selected bucket. When the user select the workflow it will be shown all existing revisions of the workflow. And finally there is a description about workflow revision.
+
+=== Studio
+Inside Studio portal there is present a `Catalog` menu. With help of this menu the user can directly interact from studio view with catalog storage.
+
+==== Publish current Workflow to the Catalog
+The Catalog menu will provide possibility for an user directly save inside catalog the created in studio workflow: `Publish current Workflow to the Catalog`.
+In opened window for publishing the current workflow to catalog the user can choose the `Bucket` where the workflow will be stored and `commit message`.
+If the workflow with the same name is already exists in specified bucket in the catalog, it will be created the new revision of this workflow.
+We recommend to specify commit message for easier differentiation the versions of a workflow.
+
+==== Get a Workflow from the Catalog
+When the workflow is selected from catalog you have two options:
+`Open as a new Workflow`
+This option will open the workflow from catalog as new Workflow in Studio
+`Append to current Workflow`
+When the user choose this option, it's supposed that the user wants to append the workflow from catalog to workflow already opened inside studio. Please open the workflow to which you want to append before.
+
+=== Workflow Catalog Portal
+The Workflow Catalog Portal provides interfaces for easy administering the catalog. The most important of provided features will be described next.
+
+==== Creating the new buckets
+You need to press `plus` button from above of buckets. In opened window you need to specify Bucket name and associate with user group.
+For security reason there are groups associated to buckets. The user from same group can view the content of bucket.
+There is a special default group - `no group` in the dropdown list of window. The bucket associated with this default group can be viewed by any user.
+
+==== Removing workflows inside bucket
+For removing workflows, first the user should select the bucket containing the workflows. Then the user can select multiple workflows with pressed CTRL button. To delete the selected workflows push the `trash` button.
 
-Evaluation priority is respected through operator precedence (`AND` over `OR`), so the following statement:
+===== Importing/Exporting Workflows into the Catalog
+In the Workflow Catalog, it is possible to Export Workflows as an archive, then Import them.
+You can export Workflows into an archive, which can be transferred by any file transfer method (email, usb key, etc.) and then imported into the other Scheduler.
```

Comment:
You can export workflows to an archive, to share them more easily (email, usb key,..), and then import the entire archive from the catalog.